### ISSUE TITLE
Rake deploy - Empty index.html

### ISSUE THREAD

**Issue by dazinator** [octopress/docs#48]
So I am the proud owner of an Octopress powered blog - hosted on GitHub here: https://github.com/dazinator/dazinator.github.io

Came accross a problem today though, hoping I can get some guidance.

```
git checkout source
rake generate
rake deploy
```

The master branch gets updated, but all of the Index.html files in master are completely empty. 

However, in back in source, If I do a `rake preview`, the contents of my "public" folder all look correct and there are Index.html files all populated and all in their correct places. I can browse the site locally via my browser and all looks correct.

Any ideas or pointers would be much appreciated - I am completely at a loss..

@JakeGinnivan - have you come accross this one before during your octopress travels?

In order to workaround this at present I am having to:-
1. In Source - `Rake Generate` - take a copy of the public folder.
2. Checkout Master
3. Copy all of the Index.html files from my copy of the Source Public folder, into the correct places in master. 
4. Git commit -a on master.

So basically, I am having to manually copying all generated index html's files to master and commit them at present :-(

You can't see this but I am openly weeping
